### PR TITLE
Feature a

### PR DESCRIPTION
**Pull request by williamWilliam10**
Pleasse you can and  help me to reviem my code

### CODE REVIEW

File: hello.py
Comment (lsrsa) [inline hello.py:1]: ok

Comment (lsrsa) [reply hello.py:1]: alpha c'est bon?

Comment (williamWilliam10) [reply hello.py:1]: please alpha respond to us as soon as possible

Comment (bricetc) [reply hello.py:1]: Hello guy, it's cool

**Comment by bricetc** [comment]
Nice to work with  you guys

**Comment by Ralph197-hub** [comment]
oui ça marche

**Pull request merged**